server/src/app.js

```diff
@@ -9,8 +9,8 @@ var indexRouter = require('./routes/index');
 var app = express();
 
 // view engine setup
-app.set('views', path.join(__dirname, 'views'));
-app.set('view engine', 'pug');
+// app.set('views', path.join(__dirname, 'views'));
+// app.set('view engine', 'pug');
```

Comment:
Why are you guys installing pug or setting up view engine?

Reply:
Have consulted team, removed the view engine. Please re-review.